changelog.yaml

```diff
@@ -18,7 +18,16 @@
 # Do not change the order ("NEXT" needs to be on top).
 - version: "NEXT"
   date: TBD
-  
+  changes:
+  - type: internal
+    impact: patch
+    title: Fix clustertask
+    description: >-
+      Fix variables in clustertask to changed structure.

```

Comment:
The impact (or not) of this "bug" on older, released versions is unclear.

Reply:
As it is already required to upgrade tekton to v0.14.3 with steward version 0.5.0 there is no additional impact.